src/objective-c/tests/BUILD

```diff
@@ -263,6 +264,20 @@ tvos_unit_test(
     ],
 )
 
+# TvTests is not runnable with bazel, so the best we can do is to test
+# that the testsuite builds correctly. This is achieved by having
+# a dummy sh_test depend on :TvTests.
+grpc_sh_test(
```

Comment:
reason why we don't just use [tvos_build_test](https://github.com/bazelbuild/rules_apple/blob/master/doc/rules-tvos.md#tvos_build_test) for this ?

Reply:
That's what I tried first and I wasn't able to get it working. It can be done as a followup if you know how.